### ISSUE TITLE
An Actual Useful Feature (Enable The Entity Radar)

### ISSUE THREAD

**Issue by The-Enigmatic-Negus** [ChristopherHaws/mc-xaero-map-spigot#6]
It would be cool if this could activate the Entity Radar for servers because it's kind of a useless plugin right now and there's kinda no point to adding players to the entity radar especially when it's only going to work for singleplayer? I mean who even uses lan servers.

**Comment by ChristopherHaws** [comment]
This plugin is only able to use events which the Xaero mod exposes. Currently, to my knowledge, the only server event which Xaero's mod listens to is world change events to get the unique level id which is what this plugin enables for Spigot.

I do think that Xaero's open parties and claims mod does have server events for this. At one point I tried to add this feature it was not very straightforward and would require the open parties and claims mod to be installed.

If Xaero adds a server event for this, I would be happy to add the feature to this plugin, but until then, sadly it isn't possible. At one point I asked if he would think about open sourcing his mod so people could contribute features but he wants to keep Xaero's closed source (from the last I have heard).

Here are some related threads where I discussed some of this with Xaero:
https://legacy.curseforge.com/minecraft/mc-mods/xaeros-world-map/issues/351
https://legacy.curseforge.com/minecraft/mc-mods/xaeros-world-map/issues/527
